app/javascript/enableSubmitOnInput.js

```diff
@@ -0,0 +1,22 @@
+const enableSubmitOnInput = () => {
+  const form = document.querySelector(".ai-annotation-form")
+  if (!form) return
+
+  const textInput = document.querySelector(".text-input")
+  const promptInput = document.querySelector(".prompt-input")
+  const submitButton = document.querySelector(".submit-button")
+
+  const toggleSubmitButton = () => {
+    if (form.checkValidity()) {
+      submitButton.disabled = false
+    } else {
+      submitButton.disabled = true
+    }
+  }
```

Comment:
このくらいの長さなら、次のように書いた方がわかりやすそうです。

```suggestion
  const toggleSubmitButton = () => submitButton.disabled = !form.checkValidity()
```

あと、書き直して見たらtoggleじゃないことに気がつきました。
enable辺りだと思います。

Reply:
関数名と中身を修正しました。
https://github.com/yush-nh/textae-ai/pull/12/commits/c52d9969d1918345db66a156bfc45252ca3c29eb